src/PrMerger.js

```diff
@@ -6,6 +6,18 @@ const GH = require('./GitHubUtil.js');
 const Util = require('./Util.js');
 const MergeContext = require('./MergeContext.js');
 
+class PrScanResult {
+    constructor() {
+        this.scanDate = new Date(); // the scan starting time
+        this.delay = null; // the next scan should start in this number of milliseconds
+        this.unchangedPrs = []; // PRs that may be ignored in the next scan
+    }
+
+    isStillUnchanged(pr) {
+        return this.unchangedPrs.some(el => el.number === pr.number && el.updated_at === pr.updated_at);
```

Comment:
Please refactor to apply one hour delta from the original issue specs: PRs that were updated delta milliseconds ago should be considered "potentially changed" even if their updated_at field has not changed. This is done to avoid ignoring same-timestamp updates. BTW, if there is a monotonically increased field that we can use instead of updated_at here, we should use that field and avoid this delta.

AFAICT, you can hard-code that delta here, in this method; no need for prUnchangedDurationMin().